velox/exec/PartitionedOutputBufferManager.cpp

```diff
@@ -146,6 +146,22 @@ void releaseAfterAcknowledge(
 }
 } // namespace
 
+PartitionedOutputBuffer::PartitionedOutputBuffer(
+    std::shared_ptr<Task> task,
+    bool broadcast,
+    int numDestinations,
+    uint32_t numDrivers)
+    : task_(std::move(task)),
+      broadcast_(broadcast),
+      numDrivers_(numDrivers),
+      maxSize_(task_->queryCtx()->config().maxPartitionedOutputBufferSize()),
+      continueSize_((maxSize_ * kContinuePct) / 100) {
```

Comment:
Would it make sense to make kContinuePct configurable? If not, perhaps, update documentation for maxPartitionedOutputBufferSize to explain that it is used together with this hard-coded setting.